packages/cyberstorm/src/components/PackageSearch/PackageSearch.tsx

```diff
@@ -53,20 +55,29 @@ export function PackageSearch(props: Props) {
 
       <div className={styles.contentWrapper}>
         <div className={styles.sidebar}>
-          <SectionMenu
-            allSections={allSections}
-            selected={section}
-            setSelected={setSection}
-          />
+          <Suspense
+            fallback={range(1, 6).map((x) => (
+              <SkeletonBox key={x} className={styles.sideBarSkeletonBar} />
+            ))}
+          >
+            <SectionMenu
+              allSections={allSections}
+              selected={section}
+              setSelected={setSection}
+            />
 
-          <CategoryMenu categories={categories} setCategories={setCategories} />
+            <CategoryMenu
+              categories={categories}
+              setCategories={setCategories}
+            />
 
-          <OthersMenu
-            deprecated={deprecated}
-            setDeprecated={setDeprecated}
-            nsfw={nsfw}
-            setNsfw={setNsfw}
-          />
+            <OthersMenu
```

Comment:
Regardless of whether `SectionMenu` and `CategoryMenu` remains inside the `Suspense`, `OthersMenu` doesn't depend on any fetched data, so it doesn't need to be wrapped inside the `Suspense`. Unless we deliberately want to hide it until the other menus are loaded, that is. Could be worthwhile to have a discussion and come up with a site-wide practice for this.

Reply:
I've went with the thought of; Suspense it if it loads something.

So with that, you're right in that it can be left outside. We can revisit this, when reviewing the page 👍